lib/caching.js

```diff
@@ -365,6 +365,14 @@ var caching = function(args) {
         self.keys = self.store.keys.bind(self.store);
     }
 
+    /**
```

Comment:
I don't think we should add this method here because it's not a common across various caches. E.g., redis uses `DBSIZE`, memcache uses `stats items`, etc.  I think this is a case where it's better to call the method directly on the store's underlying cache.  E.g., `memoryCache.store.keyCount()`, etc.

Reply:
done